babel.config.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  presets: ['next/babel'],
+  plugins: [
+    [
+      'transform-imports',
+      {
+        '@material-ui/core': {
+          // eslint-disable-next-line no-template-curly-in-string
+          transform: '@material-ui/core/${member}',
+          preventFullImport: true
+        },
+        '@hurumap-ui/charts': {
+          // eslint-disable-next-line no-template-curly-in-string
+          transform: importName =>
+            importName.toLowerCase().includes('chart')
+              ? `@hurumap-ui/charts/${importName}`
+              : `@hurumap-ui/core/${importName}`,
+          // Whether or not to throw when an import is encountered which would cause the entire module to be imported.
+          preventFullImport: true
+        },
```

Comment:
Lets add a story in PT for `@hurumap-ui/core` to export `charts` (similar to how `@material-ui/core` exports `styles`) so that we don't need this function here.

Reply:
hurumap-ui /core exports charts

Reply:
That's cool @karimkawambwa ... Then can we get away with just:
```js
          transform: '@hurumap-ui/core/${member}',
```
?

Reply:
@kilemensi 

No as mentioned case with importing `@material-ui/core/styles` that still goes to the index.js which has all other exports and defeats the whole purpose of reducing bundle size. We must get the file we want to bundle so we must use `@hurumap-ui/charts/${member}`. 

`@hurumap-ui/core/${chartMember}` will bundle everything else exported in this index file which increases the bundle sizes.

Reply:
@kilemensi 

```
export { default as Card } from './Card';
export { default as HURUmapChart } from './Card/HURUmapChart';
export { default as FlourishChart } from './Card/FlourishChart';

export { default as DropDown } from './DropDown';
export { default as EmbedDropDown } from './EmbedDropDown';

export { default as Snippet } from './Snippet';
export { default as ChartContainer } from './ChartContainer';
export { default as InsightContainer } from './InsightContainer';

export { default as A } from './A';
export { default as BlockLoader } from './BlockLoader';
export { default as ContentLoader } from './ContentLoader';
export { default as ShareDropDown } from './ShareDropDown';
export { default as TypographyLoader } from './TypographyLoader';

export { default as useProfileLoader } from './useProfileLoader';
export { default as useVisualLoader } from './useVisualLoader';

export * from '@hurumap-ui/charts';
```